mkdocs.yml

```diff
@@ -24,6 +24,7 @@ pages:
     - Home: 'en/index.md'
     - Building the firmware: 'en/build.md'
     - Flashing the firmware: 'en/flash.md'
+    - Compiling Lua:  'en/compiling.md'
```

Comment:
I can't find this file in the commit.

Reply:
That's  because I forgot to add it to the commit :astonished: